src/ansys/fluent/core/solver/flobject.py

```diff
@@ -177,11 +177,11 @@ def __init__(self, name: Optional[str] = None, parent=None):
         if name is not None:
             self._setattr("_name", name)
 
-    def set_flproxy(self, flproxy):
+    def _set_flproxy(self, flproxy):
```

Comment:
This change will require some Fluent code change in the new PyConsole. I think, at this stage, we should avoid any PyFluent change that will require sync changes in Fluent. Please also test this PR with the new PyConsole in Fluent to catch any other issue.

Reply:
Checked this PR in PyConsole. Working fine.

Reply:
'fluent_connection' and  'set_flproxy' will be handled as part of separate PRs.